README.md

```diff
@@ -24,6 +24,8 @@ Platform | Name |Description
 `sensor` | `heart_points_daily` | [Heart Points][heart-points] earned. Reset daily.
 `sensor` | `height` | [Height][height] (metres).
 `sensor` | `weight` | [Weight][weight] (kilograms).
+`sensor` | `fat` | [Body Fat][fat] (percentage).

```

Comment:
You need to add relevant links for the new sensors so [fat] actually links to something. See what's been done for the others at the bottom of this file

Reply:
Done https://github.com/YorkshireIoT/ha-google-fit/pull/74/commits/1318b9b7f971cf9b02c1b730210bd9c61309bff8